### PR TITLE
updated starterkit + fixes

### PR DESCRIPTION
**Pull request by eTorAken**
- updated es6-starterkit
- fixed linting
- updated dependencies (except react)
- improved README readability

### CODE REVIEW

File: src/Nexus.js [hunk 1]
```diff
@@ -185,6 +185,6 @@ const Nexus = {
   },
 };
 
-Nexus.Mixin = Mixin(Nexus);
+Nexus.Mixin = new Mixin(Nexus);
```
Comment (elierotenberg) [inline src/Nexus.js]: The semantics are wrong, Mixin should be renamed createMixin and called without `new`.

File: package.json [hunk 2]
```diff
@@ -17,23 +17,22 @@
...
   },
   "dependencies": {
-    "babel": "^4.3.0",
-    "bluebird": "^2.9.10",
-    "lodash": "^3.2.0",
-    "nexus-flux": "^2.0.1",
+    "babel": "^5.1.13",
+    "bluebird": "^2.9.24",
+    "lodash": "^3.7.0",
+    "nexus-flux": "^3.2.0",
```
Comment (eTorAken) [inline package.json]: @elierotenberg which version should I insert here please? previous was `2.0.1`

Comment (elierotenberg) [inline package.json]: 3.2.0, 3.3.0 was a miss.

Comment (elierotenberg) [inline package.json]: nvm 3.3.0 is actually the right version

Comment (eTorAken) [inline package.json]: I guess this version is to be published sooner or later. It is not available right now when I try to `npm install nexus-flux@3.3.0`

**Pull request merged**